src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int COLORS_COUNT = 10;
```

Comment:
You can get `.values()` from your Enum `Color` and then get `length` of returned array of values.